lib/presenter/settings.dart

```diff
@@ -1,24 +1,39 @@
-import 'package:add_2_calendar/add_2_calendar.dart';
+import 'package:device_calendar/device_calendar.dart';
 import 'package:easy_study/database/db_helper.dart';
 import 'package:easy_study/model/subject.dart';
 import 'package:flutter/material.dart';
 
-class Settings extends StatelessWidget {
+class Settings extends StatefulWidget {
+  Settings();
+
+  @override
+  SettingsPageState createState() {
+    return new SettingsPageState();
+  }
+}
+
+class SettingsPageState extends State<Settings> {
+  DeviceCalendarPlugin _deviceCalendarPlugin;
+
+  List<Calendar> calendars;
+
+  SettingsPageState() {
+    _deviceCalendarPlugin = new DeviceCalendarPlugin();
+  }
+
+  @override
+  initState() {
+    super.initState();
+    _retrieveCalendars();
+  }
+
   @override
   Widget build(BuildContext context) {
+    List<Subject> events = new List<Subject>();
     final Future<List<Subject>> subjects = DBHelper().getSubjects();
```

Comment:
you should use the dbHelper object of the Appstate instead of calling the factory DBHelper().